services/web/server/src/simcore_service_webserver/garbage_collector/plugin.py

```diff
@@ -48,3 +51,5 @@ def setup_garbage_collector(app: web.Application) -> None:
     # SEE https://github.com/ITISFoundation/osparc-issues/issues/705
     wait_period_s = settings.GARBAGE_COLLECTOR_PRUNE_APIKEYS_INTERVAL_S
     app.cleanup_ctx.append(create_background_task_to_prune_api_keys(wait_period_s))
+
+    app.cleanup_ctx.append(create_background_task_to_prune_trash(wait_period_s))
```

Comment:
Idea: At some point, we could refactor the garbage collection (GC) and use the newer approach of background tasks connected with Redis. This would make the GC scalable, and we could reuse the logic of the unique background task.

Reply:
:+1:

Reply:
totally agree! https://github.com/ITISFoundation/osparc-simcore/issues/6592